app/views/profile/partials/_form_profile.blade.php

```diff
@@ -1,7 +1,7 @@
   {{-- Birthdate --}}
   <div class="field-group -dual -alpha {{ setInvalidClass('birthdate', $errors) }}">
     {{ Form::label('birthdate', 'Birthdate: ') }}
-    {{ Form::input('date', 'birthdate', null, ['placeholder' => 'MM/DD/YYYY']) }}
+    {{ Form::text('birthdate', null, ['placeholder' => 'MM/DD/YYYY']) }}
```

Comment:
that makes sense. everything I've found says that you can't remove it while keeping the type as  date :(